### PR TITLE
Bug 1079244 - Do not show cancel dialog prompt when clicking "Uncompress...

### PR DESCRIPTION
**Pull request by mikehenrty**
...ing..." notification

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=877972875c5ebbf96544cc1cede427007f4280d6

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2976981af71bff36b27507579d8c0416b34dc3eb

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dcd924eb80cbdbddd27b801eff152092c6b642b0

File: apps/system/test/unit/update_manager_test.js [hunk 1]
```diff
@@ -1244,6 +1244,20 @@ suite('system/UpdateManager', function() {
       });
     });
 
+    suite('cancel prompt continued', function() {
+      setup(function() {
+        UpdateManager._downloading = true;
+        UpdateManager._uncompressing = true;
```
Comment (etiennesegonzac) [inline apps/system/test/unit/update_manager_test.js]: Please avoid setting internal variables as part of the setup, 
https://github.com/mikehenrty/gaia/blob/bug-1079244-fota-no-stop/apps/system/test/unit/update_manager_test.js#L603-605
should do the trick

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f1363bbecf5afbbc117a38a2e5cccc7546dfeb8b

**Pull request merged**